filedistribution/src/main/java/com/yahoo/vespa/filedistribution/FileReceiver.java

```diff
@@ -115,6 +115,7 @@ File close(long hash) {
                     File decompressedDir = Files.createTempDirectory(tmpDir.toPath(), "archive").toFile();
                     log.log(LogLevel.DEBUG, "Archived file, unpacking " + inprogressFile + " to " + decompressedDir);
                     CompressedFileReference.decompress(inprogressFile, decompressedDir);
+                    Files.delete(inprogressFile.toPath());
```

Comment:
should there be a try catch finally ?

Reply:
Yes, that's probably better, I'll look at it